src/test/java/com/jcabi/jdbc/JdbcSessionITCase.java

```diff
@@ -91,6 +98,84 @@ public void changesTransactionIsolationLevel() throws Exception {
     }
 
     /**
+     * JdbcSession can run a function (stored procedure) with
+     * output parameters.
+     * @throws Exception If something goes wrong
+     */
+    @Test
+    public void callsFunctionWithOutParam() throws Exception {
+        final DataSource dsrc = JdbcSessionITCase.source();
+        new JdbcSession(dsrc).autocommit(false).sql(
+            "CREATE TABLE IF NOT EXISTS users (name VARCHAR(50))"
+        ).execute().sql("INSERT INTO users (name) VALUES (?)")
+        .set("Jeff Charles").execute().sql(
+            StringUtils.join(
+                "CREATE OR REPLACE FUNCTION getUser(username OUT text)",
```

Comment:
@amihaiemil I know this is just a test, but we try to avoid any kind of methods with `get` as much as possible, even as a test data :) could be just named `user` I think